pages/solid-events.md

```diff
@@ -22,7 +22,7 @@ Are you interested in speaking about what you have been building on Solid? Reach
 | Date | Event | Organiser | Recordings, Slides and Notes |
 |----|-------|-------------|------------------------------|
 | 2021-04-01 | [Solid World April](https://www.eventbrite.com/e/solid-world-april-2021-tickets-144512880917)|[Solid Team](https://solidproject.org/team)||
-| 2021-03-04 | [Solid World March](https://www.eventbrite.com/e/solid-world-tickets-135552054845)|[Solid Team](https://solidproject.org/team)||
+| 2021-03-04 | [Solid World March](https://www.eventbrite.com/e/solid-world-tickets-135552054845)|[Solid Team](https://solidproject.org/team)|Community Spotlights: [Virginia Balseiro](https://www.virginiabalseiro.com/) who created [Women of Solid](https://www.womenofsolid.org/) and [Digita](https://www.digita.ai/) for their [introduction videos](https://www.youtube.com/channel/UCA22hu-0VEHt5tCc7jad74g). Ruben Verborgh gave an update on Community Solid Server (CSS) (mentioned links: [his chat from his pod](https://css.verborgh.org/public/2021/Solid World/Chat/)). Osmar Olivo asked for [feedback[(https://inrupt.atlassian.net/servicedesk/customer/portal/7) on [Pod Spaces](https://docs.inrupt.com/pod-spaces/). [LifeScope](https://lifescope.io/) presented what they're working on ([slides](https://docs.google.com/presentation/d/1vg9y_o0KtaC1j9w6yxpukEeoOsdbMCQa2zh1hQO2pYw)), and so did Creafree ([slides](https://ln2.sync.com/dl/53b6274d0/fcs466su-bs8bgykx-hkfmpz2a-rmwamczt/view/default/9895814900003)). Attendant Maria Dimou took notes: https://indico.cern.ch/event/1015408/|
```

Comment:
Whoops, I made some mistakes in the suggestion (that I also immediately applied) :sweat_smile: 

```suggestion
| 2021-03-04 | [Solid World March](https://www.eventbrite.com/e/solid-world-tickets-135552054845)|[Solid Team](https://solidproject.org/team)|Community Spotlights: [Virginia Balseiro](https://www.virginiabalseiro.com/) who created [Women of Solid](https://www.womenofsolid.org/) and [Digita](https://www.digita.ai/) for their [introduction videos](https://www.youtube.com/channel/UCA22hu-0VEHt5tCc7jad74g). Ruben Verborgh gave an update on Community Solid Server (CSS) (mentioned links: [his chat from his pod](https://css.verborgh.org/public/2021/Solid World/Chat/)). Osmar Olivo asked for [feedback](https://inrupt.atlassian.net/servicedesk/customer/portal/7) on [Pod Spaces](https://docs.inrupt.com/pod-spaces/). [LifeScope](https://lifescope.io/) presented what they're working on ([slides](https://docs.google.com/presentation/d/1vg9y_o0KtaC1j9w6yxpukEeoOsdbMCQa2zh1hQO2pYw)), and so did Creafree ([slides](https://ln2.sync.com/dl/53b6274d0/fcs466su-bs8bgykx-hkfmpz2a-rmwamczt/view/default/9895814900003)). Attendant Maria Dimou took [notes](https://indico.cern.ch/event/1015408/)|
```